src/content/docs/en/guides/integrations-guide/lit.mdx

```diff
@@ -126,6 +126,34 @@ import { MyElement } from '../components/my-element.js';
 Lit requires browser globals such as `HTMLElement` and `customElements` to be present. For this reason the Lit renderer shims the server with these globals so Lit can run. You *might* run into libraries that work incorrectly because of this.
 :::
 
+### Use TypeScript
+
+If you want use Lit with TypeScript, add the following to your `tsconfig.json` file:
```

Comment:
```suggestion
 For TypeScript support with Lit, add the following to your `tsconfig.json` file:
```

Note: if (see the last note about what this section is really about) this section is really about using decorators, then I would change this line instead to something like:

> To use experimental decorators in Lit, add the following to your `tsconfig.json` file: